src/utils/getPrereleaseFlag.ts

```diff
@@ -0,0 +1,18 @@
+import { z } from "zod";
+
```

Comment:
It might be easier to use feature flag from posthog, e.g. ```import { useFeatureFlagEnabled } from "posthog-js/react";```

Reply:
Yeah I didn't know this was an option when I created this. I'll ping you for a walkthough tomorrow